docs/run_sims.rst

```diff
@@ -154,7 +154,7 @@ List existing jobs
 
 Run ``buildstock_gcp --list_jobs your_project_file.yml`` to see a list of all existing
```

Comment:
need to tweak "see a list of all existing jobs matching" to "see the Batch job and post-processing job that matches the project specified"?